src/factories/gdaxFactories.ts

```diff
@@ -21,6 +21,19 @@ import { GDAXAuthConfig } from '../exchanges/gdax/GDAXInterfaces';
 
 let publicAPIInstance: GDAXExchangeAPI;
 
+function getAuthFromEnv(): null | GDAXAuthConfig {
+    const env = process.env;
+    if (env.GDAX_KEY && env.GDAX_SECRET && env.GDAX_PASSPHRASE) {
+        return {
+            key: process.env.GDAX_KEY,
```

Comment:
Nit: Use `env` variable

Reply:
@fb55 When you say Nit, however, the PR was merged, do you want a follow up commit?

Reply:
This was super tiny. If you want to include it in a future PR that would be great, but also not absolutely necessary.

Reply:
@fb55 Thanks. BTW, all the remaining PR you commented on should be good to go.